project-1/preprocess.ipynb

```diff
@@ -0,0 +1,524 @@
+{
```

Comment:
<p>I'm not sure if this preprocessing was used in the data that you utilized in your project because I see no exporting of the preprocessed data. If this is just experimentation, you should indicate that it is and that is if you even need to commit it anyways.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/Best_Nyah/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
actually it was for experimentation.. i'd take it out